commit e1faf20d79d63f45afd5fcdafdecc4cb59aaaf2e
Author: David Edmundson <kde@davidedmundson.co.uk>
Date:   Fri Oct 24 12:44:11 2014 +0200

    When loading the configuration set active shortcut not the default shortcut
    
    Otherwise hotkeys don't work
    
    BUG: 337230
    FIXED-IN: 5.1.1

diff --git a/libkhotkeysprivate/shortcuts_handler.cpp b/libkhotkeysprivate/shortcuts_handler.cpp
index 54aaf7c..e6c9b47 100644
--- a/libkhotkeysprivate/shortcuts_handler.cpp
+++ b/libkhotkeysprivate/shortcuts_handler.cpp
@@ -89,7 +89,7 @@ QAction *ShortcutsHandler::addAction(
         newAction->setProperty("isConfigurationAction", QVariant(true));
         }
     newAction->setText(text);
-    KGlobalAccel::self()->setDefaultShortcut(newAction, QList<QKeySequence>() << shortcut.primary());
+    KGlobalAccel::self()->setShortcut(newAction, QList<QKeySequence>() << shortcut.primary());
     // Enable global shortcut. If that fails there is no sense in proceeding
     if (!KGlobalAccel::self()->hasShortcut(newAction))
         {
